src/components/Select.jsx

```diff
@@ -217,19 +217,17 @@ export default class Select extends Component {
     let focusedOptionValue;
 
     if (has(properties, 'valueLink')) {
-      selectedValue = properties.valueLink.value;
-      focusedOptionValue = selectedValue;
+      focusedOptionValue = selectedValue = properties.valueLink.value;
```

Comment:
Agreed, I was not sure of this change. Just got motivated from underscore's code, They used it so much. Let me revert this :)